rest/rest-resources/src/main/java/org/apache/brooklyn/rest/security/provider/LdapSecurityProvider.java

```diff
@@ -69,6 +70,7 @@
     public static final Logger LOG = LoggerFactory.getLogger(LdapSecurityProvider.class);
 
     public static final String LDAP_CONTEXT_FACTORY = "com.sun.jndi.ldap.LdapCtxFactory";
+    public static final String LDAP_USER_GROUPS_ORIGIN = "brooklyn.entitlements.user.groups.origin.LdapSecurityProvider";
```

Comment:
i'd make this be `LdapSecurityProvider.class.getName()` -- there is no need for origin in the value here (it is used as a value of a key which says `origin`)

Reply:
Addressed, thanks for the feed back